x-pack/plugins/index_management/public/application/sections/home/template_list/legacy_templates/template_table/template_table.tsx

```diff
@@ -36,11 +36,73 @@ export const LegacyTemplateTable: React.FunctionComponent<Props> = ({
   history,
 }) => {
   const { uiMetricService } = useServices();
+  const {
+    core: { capabilities },
+  } = useAppContext();
   const [selection, setSelection] = useState<TemplateListItem[]>([]);
   const [templatesToDelete, setTemplatesToDelete] = useState<
     Array<{ name: string; isLegacy?: boolean }>
   >([]);
 
+  const actions: any = {
```

Comment:
I think we can use `EuiBasicTableColumn` instead of any

Reply:
Thanks, not sure how I overlooked these